MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -2,71 +2,172 @@ import UIKit
 
 final class MovieQuizViewController: UIViewController {
     // MARK: - Lifecycle
+    
+    @IBOutlet private weak var imageView: UIImageView!
+    
+    @IBOutlet private weak var counterLabel: UILabel!
+    
+    @IBOutlet private weak var textLabel: UILabel!
+    
+    private var correctAnswers:Int = 0
```

Comment:
После разделителя, перед объявлением типа - ставят пробел. Также объявление типа здесь излишне, тип может автоматически вывестись из литерала 0
```suggestion
    private var correctAnswers: Int = 0
```

Reply:
Я так меньше путаюсь, указывая тип)